### PR TITLE
Add RSS meta tag when appropriate

### LINKED ISSUE #2521: Add RSS meta tag when appropriate
# Goal

Allow rss readers to automatically add the RSS provided for each page.

# Definition of done

The generated html code for API's and Organizations should have the _link type=application/rss+xml_ meta tag.
 
More information [how to add rss autodiscovery to your site](http://www.petefreitag.com/item/384.cfm) 

# Related issues

#2310, #2478, #558, #2508

### PR DESCRIPTION
**Pull request by Vellames**
Closes #2521 

WIP

- [x] Add meta tag for specific organization
- [x] Add meta tag for specific API

### CODE REVIEW

**Comment by Vellames** [comment]
@mauriciovieira ready for review

File: apinf_packages/apis/client/lib/router.js [hunk 1]
```diff
@@ -42,6 +42,16 @@ FlowRouter.route('/apis/:slug/', {
     Meteor.call('getApi', slug, (error, api) => {
       // Check if API exists
       if (api) {
+        // Add RSS Link (only for public apis)
+        if (api.isPublic) {
```
Comment (marla-singer) [inline apinf_packages/apis/client/lib/router.js]: Combine this condition and upper as:
```js
if (api && api.isPublic) { ... }
```

Comment (Vellames) [reply apinf_packages/apis/client/lib/router.js]: @marla-singer i cant do this. 
The `if (api) {` are involving the rest of the code that contains behavious for publics and privates apis. 
Besides that, the `BlazerLayout.render` is called inside this condition, if i do the changes, all code will be called only for publics apis

Comment (marla-singer) [reply apinf_packages/apis/client/lib/router.js]: @Vellames Your truth.

**Comment by Vellames** [comment]
@mauriciovieira I did a rebase with develop branch, i tested again and the meta tag was not generated for privates apis. Can you check again please?

**Pull request merged**